src/generator/ApiGenerator.php

```diff
@@ -67,6 +67,11 @@ class ApiGenerator extends Generator
     */
     public $useJsonApi = false;
 
+    /**
+     * @var bool if true singular resource keys will be used /post/{id}, plural by default
+     */
+    public $singularResourceKeys = false;
```

Comment:
isn't this defined in the OpenAPI description?

Reply:
Do you mean - jsonApi description ?  It defined in Recommendation section https://jsonapi.org/recommendations/
so.. it looks like a recommendation only

Reply:
no, I thought it would be possible to read this key from the spec, but now I found that in spec we only have string type but not the value itself...